src/app/book/[id]/[pagenum]/page.tsx

```diff
@@ -30,6 +29,10 @@ import { MooseMilestone } from "@/components/MisconceptionComponents/MooseMilest
 import { MooseDr } from "@/components/MisconceptionComponents/MooseDr";
 import { MooseChallengingYear } from "@/components/MisconceptionComponents/MooseChallengingYear";
 import { MooseThankYou } from "@/components/MisconceptionComponents/MooseThankYou";
+import { InputActivity } from "@/components/InputActivity";
+import { TableFlowers } from "@/components/TableFlowers";
```

Comment:
This doesn't need to be different components can be one with a different config

Reply:
Try to make these all one component since they are all the same

Reply:
Done!